TradeItIosTicketSDK2/TradeItTradingTicketViewController.swift

```diff
@@ -76,11 +76,43 @@ class TradeItTradingTicketViewController: TradeItViewController, UITableViewData
 
         let isChosenAccountEnabled = self.order.linkedBrokerAccount?.isEnabled ?? false
         if (!isChosenAccountEnabled) {
-            self.accountSelectionViewController.selectedLinkedBrokerAccount = self.order.linkedBrokerAccount
-            self.navigationController?.pushViewController(self.accountSelectionViewController, animated: true)
+            self.showAccountSelection()
             return
         }
 
+        // Check if selected account supports trading equities
+        TradeItSDK.linkedBrokerManager.getAvailableBrokers(
```

Comment:
Is it possible to use the orderCapabilities on the linkedBrokerAccount ?
```swift
guard let  _ = self.order.linkedBrokerAccount?.orderCapabilities(forInstrument: TradeItTradeInstrumentType.equities)` else {
//alert
}
```

Reply:
We don't fetch orderCapabilities for equity trading and I don't think we even have implemented it yet in the API.